### DISCUSSION TITLE
Blur effect instead of just plain transparency to suite windows 11

### DISCUSSION THREAD

**Discussion by ghost** [ShankarBUS/ModernFlyouts#891] [Ideas]
There should be a blur / acrylic effect. On dialing down opacity, it just gets transparent, but it would look way better if there was a blur / 
acrylic option.
![image](https://user-images.githubusercontent.com/91146818/145705821-b40b5a2e-963e-40a2-b518-682d9ece30df.png)

**Comment by Samuel12321** [comment]
Thanks for the feedback, there have been some challenges / platform limitations which have delayed the implementation of this, but we hope to add it in the next update 0.11
https://github.com/ModernFlyouts-Community/ModernFlyouts/issues/66
